bubble.rb

```diff
@@ -1 +1,32 @@
+def bubbleSort(arr)
+  for i in 0...(arr.length - 1)
+    swapped = false
+    for j in 0...(arr.length - i - 1)
+      if arr[j] > arr[j + 1]
+        arr[j], arr[j + 1] = arr[j + 1], arr[j]
+        swapped = true
+      end
+    end
+    break if swapped == false
+  end
+  print arr
```

Comment:
Usually, you want your methods to return a value instead of sending it to the console. Change the `print arr` to just `arr`. If you wanted to print the result you can do:
```ruby
print bubble_sort_by(['hi', 'hello, 'hey']) { |left, right| left.length - right.length }
```